### DISCUSSION TITLE
Create new field on O2M

### DISCUSSION THREAD

**Discussion by Nitwel** [directus/directus#3095] [Feature Requests]
It would be nice to have an option in O2M so you can easily create a field in the target collection without needing to close the modal.

**Comment by rijkvanzanten** [comment]
Could you elaborate what you mean by

> It would be nice to have an option in O2M so you can easily create a field in the target collection

? Adding any other fields in the related collection?

**Reply by rijkvanzanten** [reply]
Ohh, are you saying to auto-generate the corresponding m2o field?

**Reply by benhaynes** [reply]
That's how I understood it... which we do have an option for, right?

**Reply by rijkvanzanten** [reply]
It's—like auto-generate for m2m—on the to-do list, yeah. I think I might've read this more like "It would be nice to manage the fields in the related collection without having to navigate to the related collection". Maybe our discussions around that from last week spoiled my 🧠 @benhaynes 😉

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
